src/components/transformation/create/GitPodButton.tsx

```diff
@@ -228,13 +229,12 @@ function GitPodButton({
                 }
             }}
         >
-            <span>
-                {submitButtonError ??
-                    intl.formatMessage({
-                        id: 'newTransform.button.cta',
-                    })}
-            </span>
-        </LoadingButton>
+            {submitButtonError
+                ? submitButtonError
+                : intl.formatMessage({
+                      id: 'newTransform.button.cta',
+                  })}
```

Comment:
Switched to the ternary operator to be safer

Reply:
If `submitButtonError` had an explicit type annotation, the original operator would be used with a high degree of confidence. Whatever you feel comfortable with is fine with me.

Reply:
It is still against a rule though - so sometimes my IDE will complain.